lib/cli.js

```diff
@@ -17,7 +17,15 @@ const Router = require('./router');
 const gens = list(process.argv.slice(2));
 
 // Override http networking to go through a proxy ifone is configured
-require('global-tunnel-ng').initialize();
+const MAJOR_NODEJS_VERSION = parseInt(process.version.slice(1).split('.')[0], 10);
+
+if (MAJOR_NODEJS_VERSION >= 10) {
+  // `global-agent` works with Node.js v10 and above.
+  require('global-agent').bootstrap();
```

Comment:
If you are not using `global` variables, then using [`createGlobalProxyAgent`](https://github.com/gajus/global-agent#setup-proxy-using-createglobalproxyagent) would be more appropriate.

You might also want to configure [`environmentVariableNamespace`](https://github.com/gajus/global-agent#createglobalproxyagent).